packages/office-ui-fabric-react/src/components/Dropdown/Dropdown.test.tsx

```diff
@@ -161,6 +161,30 @@ describe('Dropdown', () => {
       }
     });
 
+    it('issues the onChange callback when the selected item is the same if notifyOnReselect is true', () => {
+      const container = document.createElement('div');
+      let dropdownRoot: HTMLElement | undefined;
+
+      document.body.appendChild(container);
+
+      const onChangeSpy = jasmine.createSpy('onChange');
```

Comment:
Just for consistency, could you use jest.fn() for the spy instead? (I'm honestly surprised that jasmine is in our dependency tree :))

Reply:
Sounds good, updated all of the instances in this file